honcho/export/templates/upstart/master.conf

```diff
@@ -1,10 +1,5 @@
 pre-start script
```

Comment:
Yup, makes sense. I'm just going to amend the commit, if you have no objections.